Complete first task

Pobedos opened the pull request:
- HTML and CSS draft is done

Pobedos committed:
Complete first task

- HTML and CSS draft is done

Pobedos committed:
Make card form instead of full screen

Pobedos committed:
Improve card according to design

aqus commented:
первая домашняя работа - отлично, 10 баллов.

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,136 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
```
В макете указан шрифт Helvetica

Pobedos replied:
Исправлено

Pobedos replied:
А еще вопрос. Helvetica разве не платный шрифт?

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,136 @@
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
```
почему бы сюда не добавить еще и цвет шрифта и базовый размер шрифта для основного контента? Еще можно добавить line-height

Pobedos replied:
Добавил

Pobedos replied:
На самом деле не добавил изначально, потому что по данному состоянию сайта нельзя наверняка утверждать где основной контент

aqus replied:
Согласен, но об это тоже нужно подумать. Обычно, на проекте можно спросить у дизайнера, но тут можно накинуть какой-нибудь, который по вашему мнению более подходящий. Возможно, размер с кнопки очень даже подойдет.

Pobedos replied:
Хорошо, спасибо

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,136 @@
+.preview__image {
+    width: 180px;
+    height: 180px;
+    background: url("../images/img.png") center / cover;
```
почему не добавить в теге img? Так будет проще менять, если урл картинки будет приходить с бекенда

Pobedos replied:
Да, вы правы, не подумал об этом

aqus reviewed (approved):
подправить земечания и можно мержить
2 домашнее задание принято - 9.5 баллов

Pobedos committed:
Fix minor issues

- use img instead of div
- assign initial properties to body
- change font to Helvetica
